pyrft/cluster_inference.py

```diff
@@ -137,22 +137,22 @@ def cluster_tdp(data, design, contrast_matrix, mask, n_bootstraps = 100, alpha =
     mask = mask > 0
 
     # For each cluster calculate the TDP
-    for L in np.arange(n_contrasts):
+    for l in np.arange(n_contrasts):
         # Get the clusters of the test-statistic
-        cluster_im, cluster_sizes = pr.find_clusters(pvalues[..., L], cdt, below = 1, mask = mask, min_cluster_size = min_cluster_size)
+        cluster_im, cluster_sizes = pr.find_clusters(pvalues[..., l], cdt, below = 1, mask = mask, min_cluster_size = min_cluster_size)
 
         # Obtain the number of clusters
         n_clusters = len(cluster_sizes)
         
-        for I in np.arange(n_clusters):
+        for i in np.arange(n_clusters):
             # Obtain the logical entries for where each region is
-            region_idx = cluster_im == (I+1)
+            region_idx = cluster_im == (i+1)
         
             # Compute the TP bound
-            bound = sa.max_fp(pvalues[region_idx, L], thr)
+            bound = sa.max_fp(pvalues[region_idx, l], thr)
             print(region_idx.shape)
-            print(tdp_bounds[region_idx, L].shape)
-            tdp_bounds[region_idx, L] = (np.sum(region_idx) - bound)/np.sum(region_idx)
+            print(tdp_bounds[region_idx, l].shape)
+            tdp_bounds[region_idx, l] = (np.sum(region_idx) - bound)/np.sum(region_idx)

```

Comment:
```suggestion
            tdp_bounds[region_idx, l] = (np.sum(region_idx) - bound) / np.sum(region_idx)

```

Reply:
I think this change makes things less clear.

Reply:
Well, it's not a matter of personal taste, this is how you need to write the code.
https://stackoverflow.com/questions/18171560/does-pep-8-require-whitespace-around-operators-in-function-arguments

Reply:
Okay I was confused, as these spacing things do not show up as a pep8 violation on my linter.

Reply:
Happy to change them though if they are